tests/integration/test_materialized_mysql_database/materialize_with_ddl.py

```diff
@@ -1796,7 +1796,7 @@ def mysql_settings_test(clickhouse_node, mysql_node, service_name):
         clickhouse_node.query(
             "SELECT COUNT(DISTINCT  blockNumber()) FROM test_database.a FORMAT TSV"
         )
-        == "2\n"
+        == "1\n"
```

Comment:
Hm, but why the result has changed?

Reply:
Sorry, I didn't notice that it selects `blockNumber()`, so test relies on implementation details